### ISSUE TITLE
Angel explodes when called without arguments

### ISSUE THREAD

**Issue by VanTanev** [node-organic/organic-angel#3]
```
ivan@ubuntu:~/dev/bms-api$ angel

path.js:360
        throw new TypeError('Arguments to path.join must be strings');
              ^
TypeError: Arguments to path.join must be strings
    at path.js:360:15
    at Array.filter (native)
    at Object.exports.join (path.js:358:36)
    at Object.<anonymous> (/usr/local/lib/node_modules/organic-angel/bin/angel.js:8:22)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
```

Maybe default to the help command instead?

**Comment by VanTanev** [comment]
Naive local fix:

```javascript
#!/usr/bin/env node                                                                                                                                                                                                                                      
var fs = require("fs")                                                                                                                                                                                                                                   
var path = require("path")                                                                                                                                                                                                                               
var Angel = require("../index");                                                                                                                                                                                                                         
                                                                                                                                                                                                                                                         
if(!module.parent) {                                                                                                                                                                                                                                     
  var argv = process.argv.splice(2)                                                                                                                                                                                                                      
  var dnaPath = argv[0] && path.join(process.cwd(), argv[0]) || process.cwd()                                                                                                                                                                            
  var command = argv.join(" ") || "help"                                                                                                                                                                                                                 
  fs.exists(dnaPath, function(foundDNA){                                                                                                                                                                                                                 
    var instance                                                                                                                                                                                                                                         
    if(foundDNA && path.extname(dnaPath) == ".json") {                                                                                                                                                                                                   
      argv.shift() // exclude found dna path                                                                                                                                                                                                             
      instance = new Angel(dnaPath)                                                                                                                                                                                                                      
    }                                                                                                                                                                                                                                                    
    else {                                                                                                                                                                                                                                               
      instance = new Angel()                                                                                                                                                                                                                             
    }                                                                                                                                                                                                                                                    
    instance.plasma.on("ready", function(){                                                                                                                                                                                                              
      instance.on("version", function(angel, next){                                                                                                                                                                                                      
        next(null, require(path.join(__dirname, "../package.json")).version)                                                                                                                                                                             
      })                                                                                                                                                                                                                                                 
      instance.do(command, instance.render)                                                                                                                                                                                                              
    })                                                                                                                                                                                                                                                   
  })                                                                                                                                                                                                                                                     
}                         
```

**Comment by outbounder** [comment]
closing this issue as resolved in `v0.2.3`, running 

`$ npx organic-angel`

produces the following:

```
Error: no handlers found about ''
    at module.exports.do (/home/outbounder/.npm/_npx/27919/lib/node_modules/organic-angel/lib/reactor/index.js:110:17)
```

and terminates organic-angel with non-zero status code

----

PS. angel doesn't have build-in support for generating help output therefore we can not fall back to that. To have `help` command the following should be locally installed https://github.com/outbounder/angelscripts-help

PS2. May be it is good idea to provide default command handler so that angelscripts/angelabilities modules can hook on that and provide default response?
